MinimalCirclesPath.py

```diff
@@ -64,33 +64,28 @@ def getDistance(startLat, startLong, destLat, destLong):
 
 # pathWidth is the desired scaned distance from play diameter, radius is radius of circle passed to NOTAM
 def getPath( startLat, startLong, destLat, destLong, radius, pathWidth):
-
-    # TODO For actual application you would need to capture N/S/E/W
-
     
-      
-    # gets total distance from start to finish
-    totalDistance = getDistance(startLat, startLong, destLat, destLong)
-
     # gets step distance (pythagorean theorum)
     stepDistance = 2 * sqrt((radius)**2-(pathWidth/2)**2)
 
     # gets direction
     bearing = calculateBearing(startLat, startLong, destLat, destLong)
 
+    # updates start and dest slightly to adjust for area past airports
```

Comment:
I think this comment needs some elaboration, because what I think you want to be doing and what this code actually seems to be doing aren't the same.  what does "slight ... past airports" mean?

Reply:
I can update the comment but here's the logic:
    1. move the start point so that the edge in the opposite direction of the start is at the end of the area you want covered
    2. then move the destLat/destLong by half the path width. This way you have the furthest point you want NOTAM's from

are you saying there is something else unintentionally going on?